mjcf_to_sdformat/mjcf_to_sdformat/converters/link.py

```diff
@@ -25,11 +25,12 @@
 VISUAL_GEOM_GROUP = 0
 
 
-def mjcf_geom_to_sdf(body):
+def mjcf_geom_to_sdf(body, physics, body_parent_name=None):
```

Comment:
I thought we renamed this function to `mjcf_body_to_sdf`?

Reply:
https://github.com/gazebosim/gz-mujoco/pull/48/commits/1134f6a69830fde4dc29bab9ff715b105dc4656e